package.json

```diff
@@ -25,12 +26,14 @@
         "react-dom": "^16.13.1",
         "react-json-tree": "^0.11.2",
         "react-monaco-editor": "^0.36.0",
-        "typescript-to-lua": "^0.32.2"
+        "typescript-to-lua": "^0.32.2",
+        "zlib": "^1.0.5"
     },
     "devDependencies": {
         "@ark120202/typescript-config": "^2.1.0",
         "@docusaurus/core": "^2.0.0-alpha.50",
         "@docusaurus/preset-classic": "^2.0.0-alpha.50",
+        "@types/d3": "^5.7.2",
```

Comment:
IMO it should be in regular dependencies. It doesn't really matter in the case of an application, but in libraries you have to list all `@types/` packages that are somehow mentioned in the public API as non-dev dependencies, so personally I just tend to follow that pattern in apps too.